src/vorta/views/archive_tab.py

```diff
@@ -488,15 +490,28 @@ def list_result(self, result):
             self.populate_from_profile()
 
     def refresh_archive_info(self):
-        archive_name = self.selected_archive_name()
-        if archive_name is not None:
-            params = BorgInfoArchiveJob.prepare(self.profile(), archive_name)
-            if params['ok']:
-                job = BorgInfoArchiveJob(params['cmd'], params, self.profile().repo.id)
-                job.updated.connect(self._set_status)
-                job.result.connect(self.info_result)
-                self._toggle_all_buttons(False)
-                self.app.jobs_manager.add_job(job)
+        selected_archives = self.archiveTable.selectionModel().selectedRows()
+        profile = self.profile()
+
+        name_list = []
+        for index in selected_archives:
+            name_list.append(self.archiveTable.item(index.row(), 4).text())
+
+        archive_list = (
+            profile.repo.archives.select().where(ArchiveModel.name << name_list).order_by(ArchiveModel.time.desc())
+        )
+
+        archive_names = [archive.name for archive in archive_list]
```

Comment:
Why is all of that needed?

Reply:
I pasted this stuff from other pieces of the code to get the names of the selected archives, do you have a better alternative in mind?

Reply:
[`selected_archive_name`](https://github.com/borgbase/vorta/blob/2cb9afd4d551dfa5f251d6edafb29de020a4234e/src/vorta/views/archive_tab.py#LL507C24-L507C24) works with less code.

Reply:
This does the same thing with multiple archives instead of one though, right?

Reply:
Your code starts off the same as `selected_archive_name`. Then you query the database to sort the archives. This differs from the `selected_archive_name`. Why do you do that?

Reply:
```suggestion
        profile = self.profile()

        archive_names = []
        for index in selected_archives:
            archive_names.append(self.archiveTable.item(index.row(), 4).text())
```
I should be doing this right?

Reply:
Yes, that should work.